### PR TITLE
CommonTypes - Improved optional dependencies.

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
relates to: https://github.com/demisto/etc/issues/33118

### CODE REVIEW

File: Packs/CommonTypes/pack_metadata.json [hunk 1]
```diff
@@ -12,377 +12,37 @@
...
-        "ThreatMiner": {
-            "mandatory": false,
-            "display_name": "ThreatMiner"
+        "Malware": {
+            "mandatory": true,
+            "display_name": "Malware"
         },
         "CVESearch": {
-            "mandatory": false,
+            "mandatory": true,
```
Comment (glicht) [inline Packs/CommonTypes/pack_metadata.json:36]: Are we sure this is mandatory?

File: Packs/CommonTypes/pack_metadata.json [hunk 2]
```diff
@@ -12,377 +12,37 @@
...
-            "display_name": "Whois"
+        "Compliance": {
+            "mandatory": true,
+            "display_name": "Compliance"
         },
-        "Polygon": {
-            "mandatory": false,
-            "display_name": "Polygon"
+        "SplunkPy": {
+            "mandatory": true,
```
Comment (glicht) [inline Packs/CommonTypes/pack_metadata.json:20]: Are we sure this is mandatory? It was optional before.

**Review by glicht — COMMENTED** [review]
Are you sure about the packs that are now mandatory? They were optional before.

**Comment by bakatzir** [comment]
@glicht this is commin from the reputation scripts: `cveReputation`, `SplunkPySearch`.
But you are right - it should not be mandatpry but optional. i'll update here and the sdk code accordingly.

**Review by glicht — APPROVED** [review]

File: Packs/CommonTypes/ReleaseNotes/2_8_3.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+#### CommonTypes
+Improved optional dependencies.
```
Comment (amshamah419) [inline Packs/CommonTypes/ReleaseNotes/2_8_3.md:3]:
```suggestion
Modified the optional dependencies.
```

**Pull request merged**